### DISCUSSION TITLE
Config not reading my requests_limit from settings.yaml

### DISCUSSION THREAD

**Discussion by Morgan-rees34** [microsoft/graphrag#628] [Q&A]
The config is reading everything else from the settings.yaml file, including API keys, deployments etc but not the tokens_per_minute or requests_limit from the file. However, it does read it in my embedding section, but not the general LLM section. Any advice?

**Comment by KylinMountain** [comment]
you can try pull the main branch code, it has been fixed and not released yet.
But you can try set tpm and rpm which will work.
